internal/common/filespec.go

```diff
@@ -251,15 +261,44 @@ func (this *FileSpec) copyPath(baseDir string, destDir string, onlyContents bool
 		destPath = filepath.Join(destDir, filename)
 	}
 
+	paths, err := filepath.Glob(fileSpecPath)
+	if err != nil {
+		return fmt.Errorf("Failed to resolve the path pattern '%s': '%w'.", this.Path, err)
+	}
+
+	if len(paths) == 0 {
+		return fmt.Errorf("No targets found for the path '%s'.", this.Path)
+	}
+
+	// Create a new directory if multiple paths are matched.
+	if !util.PathExists(destPath) && len(paths) > 1 {
```

Comment:
!
You have a bug that comes up because of a deficiency in your spec.
You make dest if it doesn't exist, but what if it is a file?